Amit0043 translation for English (#512)
ng-alain/delon: docs/import.en-US.md

type: Dev
---

> In addition to the `ng-zorro-antd` base component and the @delon business component, sometimes we need to reference other external class libraries, the following is how to use the rich text component [ngx-tinymce](https://github.com/cipchk/ngx-tinymce):

Reviewer: ```suggestion
In addition to the NG-ALAIN base component and the @delon business component, sometimes we need to reference other external class libraries, the following is how to use the rich text component [ngx-tinymce](https://github.com/cipchk/ngx-tinymce):
```

// #endregion
```

> > The `region: third libs` region is a coding convention for `ng-alain`, register all third-party components into the `THIRDMODULES` variable, for more coding conventions, refer to [Style Guide](/docs/style-guide).

Reviewer: ```suggestion
> The `region: third libs` region is a coding convention for NG-ALAIN, register all third-party components into the `THIRDMODULES` variable, for more coding conventions, refer to [Style Guide](/docs/style-guide).
```

**Lazy loading script**

> The above import script method will package the code directly into `scripts.js`, which will cause the `scripts.js` to become larger. `ng-alain` provides another delayed loading CDN library script solution for low-usage services. (Example: [zip](https://cdn.bootcss.com/jszip/3.1.5/jszip.min.js) compression), you can use [LazyService](/util/lazy) to delay loading remote CDN scripts.

Reviewer: ```suggestion
The above import script method will package the code directly into `scripts.js`, which will cause the `scripts.js` to become larger. NG-ALAIN provides another delayed loading CDN library script solution for low-usage services. (Example: [zip](https://cdn.bootcss.com/jszip/3.1.5/jszip.min.js) compression), you can use [LazyService](/util/lazy) to delay loading remote CDN scripts.
```
